Spanner/src/Admin/Database/V1/resources/database_admin_client_config.json

```diff
@@ -20,13 +20,18 @@
         }
       },
       "methods": {
+        "ListDatabases": {
+          "timeout_millis": 3600000,
+          "retry_codes_name": "idempotent",
+          "retry_params_name": "default"
+        },
         "CreateDatabase": {
           "timeout_millis": 3600000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "GetDatabase": {
-          "timeout_millis": 30000,
+          "timeout_millis": 3600000,
```

Comment:
@skuruppu can you confirm this and the other timeout changes?

Reply:
Yes absolutely. We made this change to get all the timeouts consistent between the various configs and to just consistently make most timeouts 1 hour.

Reply:
Great, thanks for the confirmation! I'm always concerned when I see big changes like that, like to make sure they were intended before rolling them out.